package.json

```diff
@@ -4,7 +4,8 @@
 	"private": true,
 	"scripts": {
 		"dev": "vite dev",
-		"build": "vite build",
+		"dev:gen": "license-report --output=json > ./static/licenses.json",
+		"build": "yarn dev:gen && vite build",
```

Comment:
instead of chaining you can create another script called "prebuild" that will run before every "build"

Reply:
Sadly prebuild is not working.